serialization/README.md

```diff
@@ -2,13 +2,36 @@
 
 ## Serialization info
 
-_TBD_
+This directory contains the specifications and examples for all supported SPDX formats.
+
+There is a specification markdown file for each supported format describing the serialization.
+
+A collection of elements can be serialized in multiple formats.
+
+Within the model, we have an "XCollection" which represents the common properties of the collection of elements across all data formats.
+
+We can refer to the actual serialized bytes as and Artifact in the model.
+
+A Relationship of type "XCollectionOf" can be used to relate an "XCollection" to one more serialized for of the "XCollection".
+
+Within serialized data, you may need to refer to properties in the "XCollection" - such as "where to start" in a collection.
+
+In that case, the "XCollection" will be in the serialized data with ONLY the properties not supported by the serialization format.
+
+There is at most one "XCollection" in the serialization.
+
+The serialized "XCollection" shall not contain properties where that property value can be derived from the the serialized data itself.
```

Comment:
I suggest removing this line. 
I believe that the following line covers the situation adequately:
""When serializing the physical “XCollection”, any properties of the logical element which can be represented using native mechanisms in the specific serialization format (e.g. @context prefixes in JSON-LD in place of the namespaceMap) should use these mechanisms and the remaining properties should be serialized within the “XCollection” element itself.""

This line attempting to assert the inverse has potential for misunderstanding or misuse asserting that because a consumer could "figure it out" via their own analysis some useful data (e.g., where to start) should not be provided by the producer to the consumer to make it easier for them.

Reply:
Updated per suggestion